src/commands/pod_install.yml

```
@@ -10,8 +10,15 @@ parameters:
     default: "ios"
     description: The location of the "ios" directory
 steps:
+  - restore_cache:
+      keys:
+        - cache-pods-
```

```suggestion
        - cache-pods-{{ checksum "<<parameters.pod_install_directory>>/Podfile.lock" }}-{{ .Environment.CACHE_VERSION }}
        - cache-pods-
```

I believe that you need to first list the specific cache key you want to use and then provide the one without the checksums as a backup. This will allow CircleCI to know which cache is the "correct" one, but will still fallback to another if needed. Details here:

https://circleci.com/docs/2.0/caching/#restoring-cache

We actually do this wrong in a few places in the codebase and don't provide a fallback.